### ISSUE TITLE
Provide project documentation

### ISSUE THREAD

**Issue by MosesSupposes** [cehsu/sfsbooklet#21]
This should include 
- [ ] setup

- git clone, 

- yarn install
- [ ] workflow

- rebase not merge, 
- contributors self-assign tasks, 
- authors merge their work after reviewed by others, 
- enhancement work comes after all the other work is done

**Comment by cehsu** [comment]
Good idea, also assigning this to @kimcurran as she was recently onboarded and knows what would be most helpful to document.
